### PR TITLE
feature/add customer cancel support

### PR DESCRIPTION
**Pull request by bolorundurowb**

### CODE REVIEW

File: shared/helpers/FlutterwaveHelpers.js [hunk 1]
```diff
@@ -303,6 +303,45 @@ class FlutterwaveHelpers {
...
+   * @param flutterwaveId - the flutterwave transaction id
+   * @param refundAmount - should be set for a partial refund and ignored for full refunds
+   * @returns {Promise<{refundId: *, transactionId: *, flutterwaveRef: *}|null>}
+   */
+  static async refund(flutterwaveId, refundAmount = null) {
+    const options = {
+      headers: { Authorization: `Bearer ${Config.flutterwave.secretKey}` }
+    };
+
+    const payload = {};
```
Comment (Saint-lawal) [inline shared/helpers/FlutterwaveHelpers.js:318]: we didn't add the "flutterwaveId" to the payload.

**Review by Saint-lawal — CHANGES_REQUESTED** [review]
Just 1 change request. Also, I'm guessing the merchant route isn't here yet? and that you just put the template to prepare for that?

File: shared/helpers/FlutterwaveHelpers.js (hunk 1 above)
Comment (bolorundurowb) [reply shared/helpers/FlutterwaveHelpers.js:318]: We use it in the request URL.

Comment (bolorundurowb) [reply shared/helpers/FlutterwaveHelpers.js:318]: https://github.com/coupin/coupin-api/blob/b21bf947cbc3fe4bf5e0bf288546c32cfdf4923e/shared/helpers/FlutterwaveHelpers.js#L326

**Comment by bolorundurowb** [comment]
Yeah, the merchant cancellation support would be handled separately since they have more granular cancellation support

**Review by Saint-lawal — APPROVED** [review]

**Pull request merged**